src/build_tools/hooks_hydra.py

```diff
@@ -445,6 +448,19 @@ def pre_module_hook(self, *args, **kwargs):  # pylint: disable=unused-argument
                 self.log.info("[pre-module hook] Set Slurm MPI type to: %s", slurm_mpi_type)
                 self.cfg['modextravars'].update({'SLURM_MPI_TYPE': slurm_mpi_type})
 
+        if self.name in ['ANSYS', 'FLUENT']:
+            # ANSYS versions 2022+ use Intel MPI v2021 by default
+            intel_mpi = {
+                'pmi_var': 'I_MPI_PMI',
+                'pmi_set': 'pmi2',
+                'pmi_lib': '/usr/lib64/slurmpmi/libpmi2.so',
+            }
+            slurm_mpi_type = 'pmi2'
+            self.log.info("[pre-module hook] Set MPI bootstrap for Slurm")
+            self.cfg['modluafooter'] = INTEL_MPI_MOD_FOOTER.format(intel_mpi)
```

Comment:
```suggestion
            self.cfg['modluafooter'] = INTEL_MPI_MOD_FOOTER % intel_mpi
```

Reply:
oops, this is totally wrong indeed. I wanted to do what's in https://github.com/vub-hpc/build_tools/pull/54/commits/7af26828428b53ae35404ae27d71c59d21f4674e